api/src/main/java/org/openmrs/module/referenceapplication/ReferenceApplicationConstants.java

```diff
@@ -23,4 +23,6 @@
 	
 	public static final long PROCESS_HL7_TASK_INTERVAL = 5L;
 
+	public static final String LOCATION_USER_PROPERTY_NAME = "referenceapplication.locationUserPropertyName";
```

Comment:
After looking at the other pull request, i have noticed that you expect the user property to be named as locationUuid

In such a case, of what value is this global property?

Reply:
Using this Global property(name - referenceapplication.locationUserPropertyName), I want to fetch the userProperty Name(should be value of this GP). Then need to use this userPropertyName to fetch the location from the userProperty(should be value of that userProperty).

So in our module, I'm using "locationUuuid" as the userPropertyName to store the locationUuid. So setting "locationUuid" to this GP.

Reply:
If you know that the property will be named as "locationUuuid", then of what value is the gp?

Reply:
So can we move the locationUserProperty to a global property name as "locationbasedaccess.locationUserPropertyName"? So the userProperty name also will be fetched from the GP, and use that same value to this RefApp GP as well?

Reply:
What do you think? What are the pros and cons of either side?

Reply:
I think, We have already concluded this discussion :smile: